src/components/puck/index.ts

```diff
@@ -1,4 +1,16 @@
 export * from "./atoms/index.ts";
+export { AddressComponent, type AddressProps } from "./Address.tsx";
```

Comment:
I think we should have an index.ts in the puck folder and then export everything from that file here like above